apps/discovery_streams/config/dev.exs

```diff
@@ -5,18 +5,7 @@ config :discovery_streams, DiscoveryStreamsWeb.Endpoint,
   check_origin: false,
   watchers: []
 
-config :kaffe,
-  consumer: [
-    endpoints: [localhost: 9092, kafka: 9093],
-    topics: [],
-    consumer_group: "discovery-streams",
-    message_handler: DiscoveryStreams.MessageHandler,
-    offset_reset_policy: :reset_to_latest
-  ],
-  producer: [
-    endpoints: [localhost: 9092, kafka: 9093],
-    topics: ["test"]
-  ]
+config :discovery_streams, endpoints: [localhost: 9092, kafka: 9093]
```

Comment:
is this generally a new pattern v. something like elsa_brokers? I actually mostly like this, as long as we can spread this pattern